### DISCUSSION TITLE
Appropriate Temperature Data for Thermo-Structural Analysis

### DISCUSSION THREAD

**Discussion by MarcoAldegheri** [firemodels/fds#13334] [Q&A] [answered]
Hello everyone,

I’m working on a thermo-structural analysis for a steel structure, specifically an open-section steel racking system. The scope of the analysis is to obtain the temperature distribution in the steel beams and columns. However, I’m a bit unsure about the most appropriate temperature data to use as input from the FDS software for the next stage of the thermo-structural analysis, because data output are very different.

I could obtain two temperature outputs from the FDS model:

    The surface temperature of the steel member as modeled in FDS. 
    eg: &DEVC ID='TW-PA1-L0.66', QUANTITY='WALL TEMPERATURE', XYZ=14.978,5.235107,0.666, IOR=-1/

 
    The adiabatic surface temperature at the calculation point of the solid element.
    eg: &DEVC ID='TSA-PA1-L0.66', QUANTITY='ADIABATIC SURFACE TEMPERATURE', XYZ=14.978,5.235107,0.666, IOR=-1/

Given that my focus is on an open-section steel profile, thickness = 3mm , I’m uncertain which of these two temperature readings would provide a more accurate representation for my subsequent analysis (using SAFIR).

Should I use the surface temperature of the steel member as modeled, or is the adiabatic surface temperature more relevant for this type of structure?

In the attachment, you can see the model and the burner zone of the model, with the devices marked in green. I’ve noticed there are significant differences between these two temperature parameters.

Any insights or recommendations would be greatly appreciated!
![Burner](https://github.com/user-attachments/assets/f3fdfe72-fc64-4cf6-8bdf-8c13fd1f0f65)
![model](https://github.com/user-attachments/assets/8f3a3cfe-1afd-4703-bbb5-b67f23377818)

Thank you!

**Comment by Vojta-Salek** [comment]
Hello, I guess the answer depends on what you want to achieve and how you intend to define the boundary condition in SAFIR. 

AST provides an estimate of the temperature the surface would reach if there was no conduction into the material. Basically, AST represents the highest achievable thermal exposure. It may be an interesting information for the worst case scenario. 

Standard wall temperature on the other hand gives the result of a balance on the surface including all forms of heat transfer. The wall temperature better represents the actual conditions, however, it is affected by the uncertainty associated with the solid phase heat conduction model within FDS.

For me as an amateur, I would lean towards using the wall temperature if you can directly set it as a boundary condition in SAFIR. It is also possible that SAFIR can handle (or even directly requires) the AST as an input and then estimates the actual surface temperature based on material properties. I've never used SAFIR, though, so I'm looking forward to hearing back from more knowledgeable people on this topic.

Vojta

**Reply by MarcoAldegheri** [reply]
Hello Vojta,

The thermostructural analysis program performs the thermal analysis (thermal mapping) of the section by imposing the temperature as input data, specifically the temperature at the boundary of the section heated by the fire. Several publications take AST as input, and this aspect is well established; however, I wanted to better understand the limitations of FDS in calculating WALL TEMPERATURE because I find a significant difference compared to AST. Considering that I am analyzing an obstruction made of relatively thin steel material, I was wondering what could be causing such a difference.

**Reply by Vojta-Salek** [reply]
Well, the difference between AST and WALL TEMPERATURE value is caused by neglecting heat conduction into solid, right? For a well-conducting material, such as steel, I would expect the difference to be almost always significant. As for the limitations, I see weaknesses in FDS in terms of: (default) 1D heat transfer; sometimes a bit confusing and limiting description of multiple adjacent materials using different layers, problematic description of uneven surfaces, porous materials, materials changing their structure; expressing mass loss only as a change in thickness in one direction, etc. Of course, it is also not possible to include the various changes in the material caused by mechanical stresses. Those are probably exactly the phenomena addressed by a specialized (mechanical) software. Just thinking out loud, we can delete my comments afterwards.

**Reply by mcgratta** [reply, marked as answer]
Use AST. One way to think of the AST is that it is the "effective" exposing temperature that accounts for both radiation and convection. If you put your structure in a controlled furnace, the AST would provide this effective furnace temperature.

**Reply by MarcoAldegheri** [reply]
Thank you Kevin and Vojta!

**Reply by MarcoAldegheri** [reply]
Hi everyone,

Following up on a previous post, I’d like to ask for your input regarding the use of adiabatic surface temperature (AST) probes to estimate the thermal loads on steel structures for subsequent thermostructural analysis.

FDS provides two options for implementing AST probes:

Solid Surface AST Probe, which requires the physical modeling of the structure:
&DEVC ID='SOLID', QUANTITY='ADIABATIC SURFACE TEMPERATURE', XYZ=0.0,0.0,0.0, IOR=-3 /

Gas-Type AST Probe, which does not require modeling the structure itself:
&PROP ID='GAS props', EMISSIVITY=0.85, HEAT_TRANSFER_COEFFICIENT=35.0 /
&DEVC ID='GAS', PROP_ID='GAS props', QUANTITY='ADIABATIC SURFACE TEMPERATURE GAS', XYZ=0.0,0.0,0.0, ORIENTATION=0.0,0.0,-1.0 /

In my case, I'm analyzing a very lightweight steel structure placed inside a compartment. The structure will not significantly influence the overall fire dynamics or fluid flow, so I was considering using GAS-type probes to extract the temperature data needed for a separate thermal-structural model.

One concern I have is that, while the solid surface probe calculates the convective heat transfer coefficient automatically, the GAS-type probe requires manual input for both emissivity and the heat transfer coefficient. For the emissivity, I plan to assign a value representative of the steel material. However, I’m uncertain about what value to assign for the convective heat transfer coefficient.

As a preliminary assumption, I was thinking of using 35 W/m²K, as suggested in the European standard UNI EN 1991-1-2 for advanced fire models when estimating thermal actions on structures.

Does anyone have experience or recommendations regarding the reliability of GAS-type AST probes for this kind of application? And do you think using 35 W/m²K is an acceptable assumption for typical compartment fire conditions?

Thanks in advance for your insights!

**Reply by mcgratta** [reply]
I would suggest that you consider the basic shape of your structural member and compare it to whatever shape is assumed by the standard. You can also look at a heat transfer textbook to get representative values for, say, a plate, a cylinder, etc. The value of 35 does not seem out of line for a relatively thin cylinder of beam.

**Comment by RIO4FDS** [comment]
Last time I read UNI EN 1991-1-2, I remember that those values
suggested for the heat transfer coefficient on the exposed side should
consider radiation included in the heat transfer coefficient for
convection. It is some kind of approximation to use only the gas
temperature.
You should check on that (I do not have access to it right now, my
memory could be failing or the standard could be updated).
There could be some conflict in using AST, which combines both
radiation and convection to provide the thermal exposure of surfaces
(as Kevin pointed previously).

Have you looked at FDS2FTMI?
https://drive.google.com/drive/folders/0BzJEIogPt2TyfnFERGNQNFVnWXJ4aXd5bFRhRzNDSktSN1o1dERFT3ZwaHJxNkM1T0ZMeXc?resourcekey=0-d5AdP45Gi4TWutPNUXz9rg&usp=sharing


On Wed, May 14, 2025 at 10:46 AM Kevin McGrattan
***@***.***> wrote:
>
> I would suggest that you consider the basic shape of your structural member and compare it to whatever shape is assumed by the standard. You can also look at a heat transfer textbook to get representative values for, say, a plate, a cylinder, etc. The value of 35 does not seem out of line for a relatively thin cylinder of beam.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
___________________________________
Julio Cesar Goncalves da Silva, D.Sc.
www.linkedin.com/in/juliosilvafire
sites.google.com/view/jcsilva
Mobile: +55 21 98886-1235

**Reply by MarcoAldegheri** [reply]
Thanks for your reply.

I’m not exactly sure what you mean when you say “there could be some conflict in using AST, which combines both radiation and convection to provide the thermal exposure of surfaces.”
As I understand it, AST provides a temperature that can be used as thermal input for structural analysis. It accounts for both radiation and convection, yielding a single fictitious temperature to be applied as a boundary condition on the structural profile. At least, that’s my understanding based on the FDS User Guide – Technical Reference and the publication by Wickström (Diabatic Surface Temperature for Calculating Heat Transfer to Fire-Exposed Structures).

In my view, the AST represents the maximum conceivable temperature for the structural element, and thus it can be used as input for structural software.
My goal is to obtain this thermal input (AST) without having to model the entire steel structure, so I can use a simplified geometrical model instead. In this case, I intend to use the Adiabatic Surface Temperature Gas (not the gas temperature sensor, nor the solid adiabatic surface temperature). However, AST-Gas requires a convection coefficient, which is unknown.

I was wondering about the correct usage of the following line: &DEVC ID='GAS', PROP_ID='GAS props', QUANTITY='ADIABATIC SURFACE TEMPERATURE GAS', XYZ=0.0,0.0,0.0, ORIENTATION=0.0,0.0,-1.0 / when you don't know the parameter HEAT_TRANSFER_COEFFICIENT in &PROP ID='GAS props', EMISSIVITY=0.85, HEAT_TRANSFER_COEFFICIENT=35.0 /

Thanks for the document you mentioned (I didn't have it) and it uses the AST but it doesn't show the code for it. It calculates the the total heat flux from Tast and h. It is not possible to directly interface the Fire Dynamics Simulator (FDS) with the thermomechanical software currently in use; therefore, the data exchange between the two systems must be carried out manually or through intermediate processing tools, obtaining the temperature to be imposed on the structural profile from which the thermal mapping is derived. 

Please let me know if I am overlooking anything.

Thanks

**Comment by RIO4FDS** [comment]
My fault! I missed the dates on the posts. So, I made a bit of confusion.
Let´s tackle this by steps!

You want to find a Heat Transfer Coefficient correlation to use in FDS to
extract AST (right?).
This variable (h) is dependent on fluid motion (velocity field) around the
solid.
So, you may need to find a good correlation for your type of geometry as
Kevin stated.
This h is related to velocity field and surface shape.

Then, you are probably using some heat transfer coefficient in your thermal
code to calculate the total heat flux from fire to your structure (cross
section maybe?).
Some standards apply Temperature-time curves with only one part of the
total heat flux equation (convection) and give some heat transfer
coefficients to use, stating that using them radiation effects will be
already included.
This last part is what I asked you to look at. If that is the case, maybe
your standard provided heat transfer coefficient could be a number used to
"include radiation effects".
Also look at which equation your FEM code solves to calculate the heat flux
(radiation+convection or just convection?).

What I was trying to tell you is that - again, I do not have the standard
here, so It could be updated from the last time I saw it - the heat
transfer coefficient provided in the standard could be a number to include
radiation effects using temperature time curves. Take a look if that is the
case (it is usually a footnote).
If you use an equation with radiation and convection in your FEM code with
a standard heat transfer coefficient generated to include radiation
effects, you are maybe increasing your heat from fire to structures.

In Introduction (1st page) we have:
The code FDS2FTMI is included in the FDS-SVM repository [5] under
Utilities/Structural_Interaction/fds2ftmi
So, the code is available at this address.

In this Guide, we do not have a full discussion about the heat transfer
equations, but maybe you can find some more discussion in the references.

Regards,

Julio








On Wed, May 14, 2025 at 12:21 PM MarcoAldegheri ***@***.***>
wrote:

> Thanks for your reply.
>
> I’m not exactly sure what you mean when you say “there could be some
> conflict in using AST, which combines both radiation and convection to
> provide the thermal exposure of surfaces.”
> As I understand it, AST provides a temperature that can be used as thermal
> input for structural analysis. It accounts for both radiation and
> convection, yielding a single fictitious temperature to be applied as a
> boundary condition on the structural profile. At least, that’s my
> understanding based on the FDS User Guide – Technical Reference and the
> publication by Wickström (Diabatic Surface Temperature for Calculating Heat
> Transfer to Fire-Exposed Structures).
>
> In my view, the AST represents the maximum conceivable temperature for the
> structural element, and thus it can be used as input for structural
> software.
> My goal is to obtain this thermal input (AST) without having to model the
> entire steel structure, so I can use a simplified geometrical model
> instead. In this case, I intend to use the Adiabatic Surface Temperature
> Gas (not the gas temperature sensor, nor the solid adiabatic surface
> temperature). However, AST-Gas requires a convection coefficient, which is
> unknown.
>
> I was wondering about the correct usage of the following line: &DEVC
> ID='GAS', PROP_ID='GAS props', QUANTITY='ADIABATIC SURFACE TEMPERATURE
> GAS', XYZ=0.0,0.0,0.0, ORIENTATION=0.0,0.0,-1.0 / when you don't know the
> parameter HEAT_TRANSFER_COEFFICIENT in &PROP ID='GAS props',
> EMISSIVITY=0.85, HEAT_TRANSFER_COEFFICIENT=35.0 /
>
> Thanks for the document you mentioned (I didn't have it) and it uses the
> AST but it doesn't show the code for it. It calculates the the total heat
> flux from Tast and h. It is not possible to directly interface the Fire
> Dynamics Simulator (FDS) with the thermomechanical software currently in
> use; therefore, the data exchange between the two systems must be carried
> out manually or through intermediate processing tools, obtaining the
> temperature to be imposed on the structural profile from which the thermal
> mapping is derived.
>
> Please let me know if I am overlooking anything.
>
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/13334#discussioncomment-13146696>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADA4T72ZDYHEEQ2KPDFL7LD26NNOTAVCNFSM6AAAAAB5DBGU3GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTGMJUGY3DSNQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
___________________________________
Julio Cesar Goncalves da Silva, D.Sc.
www.linkedin.com/in/juliosilvafire
sites.google.com/view/jcsilva
Mobile: +55 21 98886-1235

**Comment by RobLWI** [comment]
I like to think of AST as the instantaneous surface temperature of the steel. Imagine a very thin sheet of steel backed by a well insulating material. As it is very thin it will reach equilibrium very quickly with gas temperature and radiation. There is essentially no thermal mass and no conduction involved. This AST is what you need for the SAFIR calculation, as SAFIR will calculate the effect of specific heat capacity and conduction itself. Your AST curves will be a lot more jagged and uneven than your other temperature curves, but this will all even out. I am sure there is a paper that says somewhere that AST is the appropriate method. AST is not a maximum temperature.

Out of interest, you may want to look at the RADF option for SAFIR to more directly couple FDS with your SAFIR structural model, it does take more work, and may take longer to solve, but it automates a lot of the work. But for a beginner it may be too much. However, I imagine that such a racking system, with expected fire loads, could be greatly simplified from teh model you have here. Just model the columns right next to the fire. Also are you modelling 1D elements or shell? local buckling of flanges, combined with thermal expansion forces is likely to be the failure mode. Look at the STEELSL material if doing 1D. 

You will need more AST points than in your model above, at least one each side of the columns. 

Also, I have recently performed some analysis of steel structures in fires of ~4mm steel that is directly above a fire. Comparing the temperatures calculated by FDS (exposed), and those calculated in SAFIR using AST and RADF and had very good correlation (within 5°C) . So if your materials are set up correctly in FDS, FDS wall temperatures are a good reference point for your SAFIR calculation. (But as soon as you have more complicated materials like concrete, plaster or similar, or self shading that you have not properly modelled with sufficient AST points, then the correlation will be less likely.) Based on this, it could even be argued that you could skip the SAFIR thermal calculation totally in some situations and manually edit your TSH files based on the FDS calculated temperatures, to give the appropriate temperature curve for each shell element. This may for a case like yours be quicker/easier and with better overview and less chance for getting files mixed up.
